app/static/js/app.js

```diff
@@ -325,6 +325,11 @@ document.getElementById("fullscreen-btn").addEventListener("click", (evt) => {
 document.getElementById("paste-btn").addEventListener("click", () => {
   document.getElementById("paste-overlay").show = true;
 });
+
+document.getElementById("remote-screen").addEventListener("click", () => {
```

Comment:
Could the event registration live within `remote-screen.html`?

Also, would it make more sense to react to [touch events](https://developer.mozilla.org/en-US/docs/Web/API/Touch_events/Using_Touch_Events) instead of mouse events?

Reply:
Moved event to remote-screen.html but I couldn't get the touch events to work, the input gets focus but the keyboard wouldn't show for me.